tripper/dataset/dataset.py

```diff
@@ -0,0 +1,942 @@
+"""Module for documenting datasets with Tripper.
+
+The dataset documentation follows the DCAT structure and is exposed as
+Python dicts with attribute access in this module.  This dict
```

Comment:
I suggest adding a link to DCAT here, and also explicitly stating that the statements included in the context.json are supported (if I have understood it correctly).

Reply:
done